.github/workflows/github-actions.yml

```diff
@@ -29,121 +29,27 @@ jobs:
-            toxpython: 'python3.9'
-            python_arch: 'x64'
-            tox_env: 'py39,report,codecov'
-            os: 'ubuntu-latest'
-          - name: 'py310-cover (ubuntu) + build'
-            python: '3.10'
-            toxpython: 'python3.10'
-            python_arch: 'x64'
-            tox_env: 'py310,report,codecov,build'
-            os: 'ubuntu-latest'
-          - name: 'py311-dev-cover (ubuntu)'
-            python: '3.11-dev'
-            toxpython: 'python3.11'
-            python_arch: 'x64'
-            tox_env: 'py311,report,codecov'
-            os: 'ubuntu-latest'
-
-          # windows
-          - name: 'py37-cover (windows)'
-            python: '3.7'
-            toxpython: 'python3.7'
-            python_arch: 'x64'
-            tox_env: 'py37,report,codecov'
-            os: 'windows-latest'
-          - name: 'py38-cover (windows)'
-            python: '3.8'
-            toxpython: 'python3.8'
-            python_arch: 'x64'
-            tox_env: 'py38,report,codecov'
-            os: 'windows-latest'
-          - name: 'py39-cover (windows)'
-            python: '3.9'
-            toxpython: 'python3.9'
-            python_arch: 'x64'
-            tox_env: 'py39,report,codecov'
-            os: 'windows-latest'
-          - name: 'py310-cover (windows)'
-            python: '3.10'
-            toxpython: 'python3.10'
-            python_arch: 'x64'
-            tox_env: 'py310,report,codecov'
-            os: 'windows-latest'
-          # TOOD: Enable after github actions fix errors
-          # - name: 'py311-dev-cover (windows)'
-          #   python: '3.11-dev'
-          #   toxpython: 'python3.11'
-          #   python_arch: 'x64'
-          #   tox_env: 'py311,report,codecov'
-          #   os: 'windows-latest'
-
-          # macos
-          - name: 'py37-cover (macos)'
-            python: '3.7'
-            toxpython: 'python3.7'
-            python_arch: 'x64'
-            tox_env: 'py37,report,codecov'
-            os: 'macos-latest'
-          - name: 'py38-cover (macos)'
-            python: '3.8'
-            toxpython: 'python3.8'
-            python_arch: 'x64'
-            tox_env: 'py38,report,codecov'
-            os: 'macos-latest'
-          - name: 'py39-cover (macos)'
-            python: '3.9'
-            toxpython: 'python3.9'
-            python_arch: 'x64'
-            tox_env: 'py39,report,codecov'
-            os: 'macos-latest'
-          - name: 'py310-cover (macos)'
-            python: '3.10'
-            toxpython: 'python3.10'
-            python_arch: 'x64'
-            tox_env: 'py310,report,codecov'
-            os: 'macos-latest'
-          - name: 'py311-dev-cover (macos)'
-            python: '3.11-dev'
-            toxpython: 'python3.11'
-            python_arch: 'x64'
-            tox_env: 'py311,report,codecov'
-            os: 'macos-latest'
-
+          - python: '3.10'
+            os: ubuntu-latest
+            tox_env: 'docs'
+          - python: '3.10'
```

Comment:
what about update-readme?

(in order to check if there is a diff)

Reply:
It was commented out in `tox.ini` and I could not find any reference to it in github-actions so I don't think it was running previously and removed it, I can enable it if you like to.

Reply:
OMG

You just find another missing thing

Yes, I think it is needed

Do the another projects has the same situation?

Reply:
`pylint-odoo` and `odoo-pre-commit-hooks` do run `update-readme` so no issues there.

I was looking into it and it seems that there is no test_build_docstring in the source code, so we will need to write it before adding it to the CI. 

This is the commit where `update-readme` was added to `tox.ini`: [39156804](https://github.com/Vauxoo/pre-commit-vauxoo/commit/39156804#diff-8b66610a43eb066fd4bbb4d99ad34425ffaec81be10d0174e86252b7a09dbe34)

It seems we forgot to add the test for it.